packages/plugin-ext/src/main/browser/view/plugin-tree-view-node-label-provider.ts

```diff
@@ -43,8 +43,11 @@ export class PluginTreeViewNodeLabelProvider implements LabelProviderContributio
             return node.icon;
         }
         if (node.themeIconId) {
-            const uri = node.resourceUri && new URI(node.resourceUri) || undefined;
-            return this.labelProvider.getIcon(URIIconReference.create(node.themeIconId, uri));
+            if (node.themeIconId === 'file' || node.themeIconId === 'folder') {
```

Comment:
Are these two special-cased so we can get stuff like the "Java" icon for *.java files?

Reply:
correct